Cardu/yelp-findo
# #3: search restaurants by location name (closes #3)

Closes #3

## Test Plan

### tests performed

manual testing: 
-launched app on localhost:3000
-launched corsproxyhttps on 1337 (see Readme)
-searched for restaurants in Milan, Rome, Milano and changing the Radius to see more results

unit tests:
-single components shallow rendering

### tests not performed (domain coverage)
-query test with mocked fetch (done fetch setup but can't make different queries work with it)
